docs-chef-io/content/inspec/resources/k8sobject.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "k8sobject resource"
+draft = false
+gh_repo = "inspec"
+platform = "k8s"
+
+[menu]
+[menu.inspec]
+title = "k8sobject"
+identifier = "inspec/resources/k8s/K8s Object"
+parent = "inspec/resources/k8s"
++++
+
+
+Use the `k8sobject` Chef InSpec audit resource to test the configuration of...
```

Comment:
I would suggest having an explanation that this is a generic InSpec resource used to test any Kubernetes object. Trailing off with an ellipsis will just add confusion.